app/src/components/AppBar/WindowsControls.tsx

```diff
@@ -1,39 +1,47 @@
-import { Cross1Icon, MinusIcon, SquareIcon } from '@radix-ui/react-icons';
+import { useState } from 'react';
+import { CopyIcon, Cross1Icon, MinusIcon } from '@radix-ui/react-icons';
 import { Button } from '../ui/button';
 import { MainChannels } from '/common/constants';
 import { WindowCommand } from '/common/models/project';
 
 export const WindowsControls = () => {
+    const [isMaximized, setIsMaximized] = useState(true);
+
     if (process.platform !== 'win32') {
         return null;
     }
 
     function sendCommand(command: WindowCommand) {
         window.api.invoke(MainChannels.SEND_WINDOW_COMMAND, command);
+        if (command === WindowCommand.MAXIMIZE || command === WindowCommand.UNMAXIMIZE) {
+            setIsMaximized(!isMaximized);
+        }
     }
 
     return (
-        <div className="flex mx-2 text-foreground-active">
+        <div className="flex text-foreground-active h-full">
             <Button
                 onClick={() => sendCommand(WindowCommand.MINIMIZE)}
                 variant={'ghost'}
-                className="hover:bg-background-onlook/30  hover:text-foreground outline-border"
+                className="hover:bg-background-onlook/30  hover:text-foreground outline-border w-full h-full rounded-none"
                 aria-label="Minimize"
             >
                 <MinusIcon className="h-3 w-3" />
             </Button>
             <Button
-                onClick={() => sendCommand(WindowCommand.MAXIMIZE)}
+                onClick={() =>
+                    sendCommand(isMaximized ? WindowCommand.UNMAXIMIZE : WindowCommand.MAXIMIZE)
+                }
                 variant={'ghost'}
-                className="hover:bg-background-onlook/30 hover:text-foreground outline-border"
+                className="hover:bg-background-onlook/30 hover:text-foreground outline-border w-full h-full rounded-none"
                 aria-label="Maximize"
             >
-                <SquareIcon className="h-3 w-3" />
+                <CopyIcon className="h-3 w-3 scale-x-[-1]" />
             </Button>
             <Button
                 onClick={() => sendCommand(WindowCommand.CLOSE)}
                 variant={'ghost'}
-                className="hover:bg-background-onlook/30 hover:text-foreground outline-border"
+                className="hover:bg-[#e81123] active:bg-[#e81123]/50 hover:text-foreground outline-border w-full h-full rounded-none"
```

Comment:
Should we consider using our tokens here or is this the standard hex color @abhiroopc84?
Particularly, we have bg-red-500 as well.

https://github.com/onlook-dev/onlook/blob/e8e06499deb4e37967f33eed730cf7a6250a7bf7/app/src/index.css#L6

Reply:
It's a standard hex color that I found across some major windows electron apps including vscode, spotify. We could use our tokens as well, it's just a personal choice I suppose. Let me know if you want me to use our tokens.

Reply:
I see, let's keep them as is. Since these are perceived as system-level control, I don't want them to be affected by us refactoring our color tokens for example. 
Thanks for the thoughtful response.